sponsorship/src/main/java/com/kyriba/conference/sponsorship/configuration/SwaggerConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package com.kyriba.conference.sponsorship.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+
+/**
+ * @author M-ASL
+ * @since v1.0
+ */
+@Configuration
+@EnableSwagger2
+public class SwaggerConfiguration
+{
+  @Bean
+  public Docket productApi()
+  {
+    return new Docket(DocumentationType.SWAGGER_2)

```

Comment:
Add global API info and apply filter at URL level

Reply:
Fixed